pibooth/printer.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, name='default', max_pages=-1):
         self.nbr_printed = 0
         if not cups:
             LOGGER.warning("No printer found (pycups or pycups-notify not installed)")
-            return  # CUPS is not installed
+            return # CUPS is not installed
```

Comment:
According PEP8, inline comment should be : 2 spaces + dash + 1 space, isn't it?

Reply:
I'm not a strict PEP8 user, so ok to put it again

Reply:
In fact I removed it while trying to remove this return (no-else-return pylint warning). But it will be in another PR.